### PR TITLE
Simplified setup and docs

### PR DESCRIPTION
**Pull request by stas**
Some suggestions and notes I tried to document or simplify while working on Aleph.

There's a new installation guide I would like to get some feedback on.
I also tried to simplify the dockerfiles where possible and reduce the footprint of running Aleph containers.

Following the guide, getting started with a local development instance now should be much easier and clear.

/cc @pudo

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -56,52 +64,57 @@ services:
         C_FORCE_ROOT: 'true'
         ALEPH_ELASTICSEARCH_URI: http://elasticsearch:9200/
         ALEPH_DATABASE_URI: postgresql://aleph:aleph@postgres/aleph
-        POLYGLOT_DATA_PATH: /polyglot
-        TESSDATA_PREFIX: /usr/share/tesseract-ocr
```
Comment (pudo) [inline docker-compose.yml]: Did you remove this intentionally? If not set, this will disable OCR.

File: docker-compose.yml [hunk 2]
```diff
@@ -56,52 +64,57 @@ services:
...
         ALEPH_BROKER_URI: amqp://guest:guest@rabbitmq:5672
-        # ALEPH_NEO4J_URI: http://neo4j:neo4j@neo4j/
+        ALEPH_NEO4J_URI: http://neo4j:neo4j@neo4j/
```
Comment (pudo) [inline docker-compose.yml]: We're not actually using Neo4J at the moment.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,6 +1,13 @@
+# We can not extend services with liks and we don't want it,
+# overall, the app image is exactly the same.
```
Comment (pudo) [inline docker-compose.yml]: I don't get this comment

File: aleph/manage.py [hunk 4]
```diff
@@ -137,12 +137,14 @@ def indexentities(foreign_id=None):
 
 
 @manager.command
-def init():
+@manager.option('-s', '--skip', dest='skip', default='')
```
Comment (pudo) [inline aleph/manage.py]: Flag name is a bit ambiguous -- perhaps ``--skip-downloads``?

File: contrib/base/Dockerfile [hunk 5]
```diff
@@ -1,35 +1,29 @@
...
         tesseract-ocr-pol tesseract-ocr-slk tesseract-ocr-slv \
         tesseract-ocr-sqi tesseract-ocr-srp tesseract-ocr-tur \
-        tesseract-ocr-ukr nodejs djvulibre-bin \
-    && apt-get -y autoremove && apt-get clean
-ENV TESSDATA_PREFIX /usr/share/tesseract-ocr
-
-RUN wget --quiet -O /tmp/kat.deb http://ftp.us.debian.org/debian/pool/main/t/tesseract-ocr-kat/tesseract-ocr-kat_3.04.00-1_all.deb \
-    && dpkg -x /tmp/kat.deb /tmp && cp /tmp/usr/share/tesseract-ocr/tessdata/kat.traineddata /usr/share/tesseract-ocr/tessdata \
-    && rm -rf /tmp/kat.deb /tmp/usr
+        tesseract-ocr-ukr nodejs nodejs-dev nodejs-legacy npm djvulibre-bin \
```
Comment (pudo) [inline contrib/base/Dockerfile]: cool!

File: docs/installation.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+`aleph.env` and follow the steps below to edit it.
+
+To get the OAuth credentials please visit the [Google Developers Console](https://console.developers.google.com/).
+There you will need to [create an API key](https://support.google.com/googleapi/answer/6158862).
+In the **Authorized redirect URIs** section, use this URL:
+`http://lvh.me:13376/api/1/sessions/callback/google`.
+Save the client ID and the client secret as `ALEPH_OAUTH_*` values.
+
+Finally you will need to provide a value for the `ALEPH_SECRET_KEY`. A good
+example of a value is the output of `openssl rand -hex 24`.
```
Comment (pudo) [inline docs/installation.md]: neat trick!

File: docker-compose.yml (hunk 3 above)
Comment (stas) [inline docker-compose.yml]: Oh, should probably remove it. It is related to the use of `image: aleph_app` below in the services.

File: docker-compose.yml (hunk 1 above)
Comment (stas) [inline docker-compose.yml]: This was moved into the aleph_base image since that is the place where the OCR was configured and installed.

**Comment by stas** [comment]
@pudo this should be ready for a final review. I will update the #112 in the meantime.

**Comment by pudo** [comment]
Thanks, this is really useful.

**Pull request merged**